src/Menu.cpp

```diff
@@ -571,18 +571,23 @@ void Menu::DrawOverlay()
 	compiledShaders = shaderCache.GetCompletedTasks();
 	totalShaders = shaderCache.GetTotalTasks();
 
+	auto state = State::GetSingleton();
+
 	auto failed = shaderCache.GetFailedTasks();
 	auto hide = shaderCache.IsHideErrors();
 	auto stats = shaderCache.GetShaderStatsString();
+	auto progressTitle = state->IsDeveloperMode() ? fmt::format("Compiling Shaders: {}", stats) : fmt::format("Compiling Shaders: {}", shaderCache.GetShaderStatsString(true).c_str());
```

Comment:
```
auto stats = ... 
auto progressTitle = ...
```
Can be simplified into 
```
auto progressTitle = fmt::format("Compiling Shaders: {}", shaderCache.GetShaderStatsString(!state->IsDeveloperMode()));
```

Reply:
Yeah 👍 
Added a PR!
https://github.com/alandtse/skyrim-community-shaders/pull/2